Makefile

```diff
@@ -11,23 +12,23 @@ LIBS = $(filter-out $(MAIN) ,$(OBJS))
 all: main testToken testParser
 
 main: $(LIBS) srcs/main.o
-	$(CPP) $(CPPFLAGS) -o $@ $^
+	$(CPP) $(CPPFLAGS) -o $@ $^ $(LD)
 
 testToken: $(LIBS) tests/testToken.o
-	$(CPP) $(CPPFLAGS) -o $@ $^
+	$(CPP) $(CPPFLAGS) -o $@ $^ $(LD)
 
 testParser: $(LIBS) tests/testParser.o
-	$(CPP) $(CPPFLAGS) -o $@ $^
+	$(CPP) $(CPPFLAGS) -o $@ $^ $(LD)
 
 srcs/%.o : srcs/%.cc $(DEPDIR)/%.d | $(DEPDIR)
-	 $(COMPILE.cc) $< -o $@
+	 $(COMPILE.cc) $< -o $@ 
```

Comment:
Trailing spaces on this line and line 27 should be removed

Reply:
Removed.